src/main/resources/templates/thymeleaf/person/detail-section/action-delete-person.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+  <head>
+    <meta charset="UTF-8" />
+    <title>person delete action</title>
+  </head>
+  <body>
+    <th:block th:fragment="section">
+      <th:block th:if="${personDeletionConfirmationValidationError}">
+        <script type="text/javascript">
+          document.addEventListener("DOMContentLoaded", function () {
+            document.querySelector("#person-delete-form")?.classList.remove("tw-hidden");
+            document.querySelector("#person-delete-description")?.classList.add("tw-hidden");
+          });
+        </script>
+      </th:block>
+
+      <div
+        th:replace="fragments/section-heading::section-heading(~{::person-details-dangerzone-body}, ~{::person-details-dangerzone-actions})"
+      >
+        <th:block th:ref="person-details-dangerzone-body">
+          <h2 th:text="#{person.account.dangerzone.title}">Benutzer löschen</h2>
+        </th:block>
+      </div>
+      <div
+        th:replace="fragments/box::box-with-icon(~{::person-details-dangerzone-box-icon}, ~{::person-details-dangerzone-box-body})"
+      >
+        <th:block th:ref="person-details-dangerzone-box-icon">
+          <div
+            th:replace="fragments/rounded::block(content=~{::person-details-dangerzone-box-icon-rounded}, className='tw-bg-red-500 tw-text-white dark:tw-bg-red-500 dark:tw-text-zinc-900')"
+          >
+            <th:block th:ref="person-details-dangerzone-box-icon-rounded">
+              <svg th:replace="icon/trash::svg(className='tw-w-8 tw-h-8')" />
+            </th:block>
+          </div>
+        </th:block>
+        <th:block th:ref="person-details-dangerzone-box-body">
+          <div id="person-delete-description">
+            <p th:text="#{person.account.dangerzone.description}"></p>
+            <button
+              class="button-danger"
+              th:text="#{person.account.dangerzone.delete-button(${person.niceName})}"
+              onclick="document.querySelector('#person-delete-form')?.classList.remove('tw-hidden');document.querySelector('#person-delete-description')?.classList.add('tw-hidden');"
```

Comment:
mit `<button formaction="beste-formaction">` kann das `form` an eine andere url ge`POST`ed werden. dann geht das ganze auch ohne JS.

und dann schauen wir wie wo was mit hotwire/turbo neu gerendert wird.

alternativ zusätzlich mit `<noscript>` das delete `form` ohne `tw-hidden` rendern. dann wäre halt dieser button hier doppelt gemoppelt und ohne js hätte man die zwei Klicks nicht. Was imho allerdings in Ordnung wäre, da die eigentliche Sicherung ja die eingabe des namens ist.